2023/refractive-index-questions/french/sentence_translations.json

```diff
@@ -56,7 +56,7 @@
  {
   "input": "And derive the formula describing exactly how much light bends.",
   "model": "nmt",
-  "translatedText": "Et comment trouver la formule décrivant avec précision l'angle de la déviation.",
+  "translatedText": "et dériver la formule décrivant la déviation exacte de la lumière.",
```

Comment:
Although "dériver une formule" exists in French, it is not commonly used and has another sense in mathematics that can be misleading, alternatives being "déduire", "déterminer", "exprimer"...

Reply:
You're right I hadn't thought about the possible confusion with 'dériver'. I don't think 'déduire' would be appropriate in this context. Of your teo other suggestion, I prefer 'déterminer', but 'obtenir' would also be a good alternative I suppose.
As a more general question regarding this whole translation project, maybe it would be better for conversion to be in the language of the translation being looked at. What do you think?